README.md

```diff
@@ -114,37 +336,47 @@ $ mbed dm init -d "<your company name.com>" --model-name "<product model identif
 
- 2. Set a global `mbed config` variable `CLOUD_SDK_API_KEY` on the host machine valid for the account that your device will connect to. For example:
+  Also, there are a number of board-specific applications that focus on providing more elaborate hardware features with Mbed OS and the Pelion IoT Platform. These are available in the Pelion [Quick-Start](https://cloud.mbed.com/quick-start).
 
-     ```mbed config -G CLOUD_SDK_API_KEY <API_KEY>```
+2. Set a global `mbed config` variable `CLOUD_SDK_API_KEY` on the host machine valid for the account that your device will connect to. For example:
+    ```
+    $ mbed config -G CLOUD_SDK_API_KEY <API_KEY>
+    ```
 
-     For instructions on how to generate an API key, please [see the documentation](https://cloud.mbed.com/docs/current/integrate-web-app/api-keys.html#generating-an-api-key).
+    For instructions on how to generate an API key, please [see the documentation](https://cloud.mbed.com/docs/current/integrate-web-app/api-keys.html#generating-an-api-key).
 
- 3. Initialize your Pelion DM credentials (once per project):
-    ```mbed dm init -d "<your organization>" --model-name "<product model>" -q --force```
+3. Initialize your Pelion DM credentials (once per project):
+    ```
+    $ mbed dm init -d "<your company name.com>" --model-name "<product model identifier>"
+    ```
 
-    This will create your private/public key pair and also initialize various .c files with these credentials, so you can use Pelion DM connect and (firmware) update features.
+    This will create your private/public key pair and also initialize various .c files with these credentials, so you can use Connect and (firmware) Update device management features.
 
- 4. Remove the `main.cpp`application from the project, or ensure the content of the file is wrapped with `#ifndef MBED_TEST_MODE`.
+4. Remove the `main.cpp` application from the project, or ensure the content of the file is wrapped with `#ifndef MBED_TEST_MODE`.
  
- 5. Compile the tests with the `MBED_TEST_MODE` compilation flag.
-    
-    ```mbed test -t <toolchain> -m <platform> --app-config mbed_app.json -n simple-mbed-cloud-client-tests-* -DMBED_TEST_MODE --compile```
+5. Compile the tests with the `MBED_TEST_MODE` compilation flag.
+
+    ```
+    $ mbed test -t <toolchain> -m <platform> --app-config mbed_app.json -n simple-mbed-cloud-client-tests-* -DMBED_TEST_MODE --compile
+    ```
 
- 5. Run the Simple Pelion Device Management Client tests from the application directory:
+5. Run the Simple Pelion Device Management Client tests from the application directory:
 
-     ```mbed test -t <toolchain> -m <platform> --app-config mbed_app.json -n simple-mbed-cloud-client-tests-* --run -v```
+    ```
+    $ mbed test -t <toolchain> -m <platform> --app-config mbed_app.json -n simple-mbed-cloud-client-tests-* --run -v
+    ```
 
 ### Troubleshooting
 Below are a list of common issues and fixes for using Simple Pelion DM Client.
```

Comment:
Simple Pelion DM Client sounds like the correctly-branded name; is that what we should use throughout this document when we review?